harper-core/src/linting/closed_compounds.rs

```diff
@@ -82,12 +82,94 @@ create_closed_compound_linter!(Northeastern, "north eastern", "northeastern");
 create_closed_compound_linter!(Upholstery, "up holstery", "upholstery");
 create_closed_compound_linter!(Proofread, "proof read", "proofread");
 
+create_closed_compound_linter!(Forever, "for ever", "forever");
+create_closed_compound_linter!(Into, "in to", "into");
+create_closed_compound_linter!(Everyone, "every one", "everyone");
+create_closed_compound_linter!(Nobody, "no body", "nobody");
+create_closed_compound_linter!(Somebody, "some body", "somebody");
+create_closed_compound_linter!(Anybody, "any body", "anybody");
+create_closed_compound_linter!(Nothing, "no thing", "nothing");
+create_closed_compound_linter!(Anyway, "any way", "anyway");
+create_closed_compound_linter!(Anytime, "any time", "anytime");
+create_closed_compound_linter!(Anywhere, "any where", "anywhere");
+create_closed_compound_linter!(Nowhere, "no where", "nowhere");
+create_closed_compound_linter!(Everywhere, "every where", "everywhere");
+create_closed_compound_linter!(Overnight, "over night", "overnight");
+create_closed_compound_linter!(Instead, "in stead", "instead");
+create_closed_compound_linter!(Backpack, "back pack", "backpack");
+create_closed_compound_linter!(Cupboard, "cup board", "cupboard");
+create_closed_compound_linter!(Keyboard, "key board", "keyboard");
+create_closed_compound_linter!(Somewhere, "some where", "somewhere");
+create_closed_compound_linter!(Notebook, "note book", "notebook");
```

Comment:
"note book" are equal variants for the book sense but it's never been used for the laptop sense.